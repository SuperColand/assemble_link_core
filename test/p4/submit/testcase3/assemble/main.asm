.data
	arr: .space 20

.text
main:
	la $t1, arr
	li $t0, 48
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 7196
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 9294
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 9091
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 7031
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 3577
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 7702
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 3503
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 7217
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 2168
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 5409
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 8233
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 2023
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 7152
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 1578
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 2399
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 3863
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 6025
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 8489
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	li $t0, 9718
	sw $t0, 0($t1)
	addiu $t1, $t1, 4
	la $a0, arr
	li $a1, 20
	jal bubble
	syscall
