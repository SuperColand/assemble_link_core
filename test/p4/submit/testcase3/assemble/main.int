.data
20

.text
lui $at arr@Hi
ori $t1 $at arr@Lo
addiu $t0 $0 48
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 7196
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 9294
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 9091
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 7031
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 3577
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 7702
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 3503
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 7217
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 2168
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 5409
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 8233
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 2023
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 7152
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 1578
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 2399
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 3863
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 6025
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 8489
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 9718
sw $t0 0 $t1
addiu $t1 $t1 4
lui $at arr@Hi
ori $a0 $at arr@Lo
addiu $a1 $0 20
jal bubble
syscall
