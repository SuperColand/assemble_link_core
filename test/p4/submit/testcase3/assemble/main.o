3c010000
34290000
24080030
ad280000
25290004
24081c1c
ad280000
25290004
2408244e
ad280000
25290004
24082383
ad280000
25290004
24081b77
ad280000
25290004
24080df9
ad280000
25290004
24081e16
ad280000
25290004
24080daf
ad280000
25290004
24081c31
ad280000
25290004
24080878
ad280000
25290004
24081521
ad280000
25290004
24082029
ad280000
25290004
240807e7
ad280000
25290004
24081bf0
ad280000
25290004
2408062a
ad280000
25290004
2408095f
ad280000
25290004
24080f17
ad280000
25290004
24081789
ad280000
25290004
24082129
ad280000
25290004
240825f6
ad280000
25290004
3c010000
34240000
24050014
0c000c43
0000000c
24080000
00058021
2610ffff
11100012
24090000
240a0000
02088823
1131000c
254b0004
008a6821
008b7021
8db20000
8dd30000
02726023
05810002
add20000
adb30000
25290001
254a0004
08000c4a
25080001
08000c46
03e00008
