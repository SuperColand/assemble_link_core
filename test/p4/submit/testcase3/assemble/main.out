.data
20

.text
3c010000
34290000
24080030
ad280000
25290004
24081c1c
ad280000
25290004
2408244e
ad280000
25290004
24082383
ad280000
25290004
24081b77
ad280000
25290004
24080df9
ad280000
25290004
24081e16
ad280000
25290004
24080daf
ad280000
25290004
24081c31
ad280000
25290004
24080878
ad280000
25290004
24081521
ad280000
25290004
24082029
ad280000
25290004
240807e7
ad280000
25290004
24081bf0
ad280000
25290004
2408062a
ad280000
25290004
2408095f
ad280000
25290004
24080f17
ad280000
25290004
24081789
ad280000
25290004
24082129
ad280000
25290004
240825f6
ad280000
25290004
3c010000
34240000
24050014
0c000000
0000000c

.symbol
0	%arr
0	main

.relocation
0	arr@Hi
4	arr@Lo
248	arr@Hi
252	arr@Lo
260	bubble
